### DISCUSSION TITLE
Is there any way to set back ground color to a ToolItem.

### DISCUSSION THREAD

**Discussion by purnac123** [eclipse-platform/eclipse.platform.swt#298] [Q&A]
In scenarios where a toolbar has multiple ToolItem controls of PUSH type, the selected ToolItem and the focus ToolItem end up having almost similar background colors. 
For accessibility needs it will be helpful if we can distinguish between the 2 colors for the ease of color blind users.
As a result a **setBackground** method similar to what we have for buttons will be very helpful.
If not, even if we can perform a **setStyle** on ToolItems to set it to bold or Italics on focus we will be able to somewhat distinguish between focus and selected controls.
Please note, I tried drawing a rectangle around the ToolItems using **GC.drawRectangle** method.
However the rectangle vanishes within seconds making it hard for the user to properly see it.

**Comment by mickaelistria** [comment]
Is there a "standard" approach that native application do use for this problem? SWT is supposed to "inherit" native accessibility from the OS graphical library, so I'm curious if SWT behaves as expected (and OS libraries also have this issue), or whether SWT is "diminished" compared to the OS backend?
